packages/components/src/components/combobox/shadow.tsx

```diff
@@ -266,9 +274,16 @@ export class KolCombobox implements ComboboxAPI {
 				handleEvent(true, () => this.moveFocus(-1));
 				break;
 			}
+			case 'Tab':
```

Comment:
Das funktioniert nur, wenn explizit Tab gedrückt wurde. Nicht für alle anderen Wege, wie der Fokus verloren gehen kann. Schau dir am besten mal an, wie das bei SingleSelect umgesetzt ist: https://github.com/public-ui/kolibri/pull/6546/files#diff-82183195a6f6176697e5362801d4f628dd0599c6f60ce3ee2ffce09a3ace4d71R636

isOpen würde ich immer auf `false` setzen, wenn der Fokus in irgendeiner Form verloren geht (einschließlich window focus). Warum wir hier `refInput.focus()` ausführen verstehe ich noch nicht.

Reply:
Ich verstehe den Ansatz einen Event-Listener zu verwenden und habe es auch versucht umzusetzen, aber das einzige Resultat welches ich damit hinbekomme, ist das, dass der Focus zu erst auf das Input zurückwandert und erst beim zweiten Tab das Dropdown schließt. `this.refInput?.focus();` sorgt derzeitig dafür das die Logik mit dem selben Tab das Input wieder fokussiert und gleichzeitig in das nächste Feld im DOM springt. Ohne diese Methode wandert der Fokus zu dem ersten Element im DOM.

Reply:
Wir hebeln ja die native Tab-Methode nicht aus, sondern erweitern diese nur